docker-compose.override.yml

```diff
@@ -1,3 +1,4 @@
+version: "3.9"
```

Comment:
```suggestion
version: "3.7"
```

This is the change to the version you needed to get it to work right?

Reply:
The version in the non-override docker compose also needs to be changed, but yes.

Reply:
To be fixed in #128